x-pack/plugins/discover_enhanced/kibana.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "discoverEnhanced",
+  "version": "8.0.0",
+  "kibanaVersion": "kibana",
+  "server": false,
+  "ui": true,
+  "requiredPlugins": ["uiActions", "embeddable", "discover"],
+  "optionalPlugins": ["share"]
+}
```

Comment:
Could you add the discover_advanced path to CODEOWNERS (team @elastic/kibana-app), many thx!

Reply:
Done.